app/assets/javascripts/discourse/app/components/sidebar/anonymous/categories-section.hbs

```diff
@@ -16,6 +15,7 @@
       @prefixValue={{sectionLink.prefixValue}}
       @prefixColor={{sectionLink.prefixColor}}
       @prefixElementColors={{sectionLink.prefixElementColors}}
+      data-category-id={{sectionLink.category.id}}
```

Comment:
Not sure if it's possible, but is there a way to do something like `@dataAttributes={{sectionLink.dataAttributes}}` which would be a hash like `{ category_id: this.category.id }`, which could be expanded into `data-X` attributes inside the `SectionLink` component?

If not that's fine, I just thought it would be handy to be able to do this like `classNames`, which for example we change dynamically in chat https://github.com/discourse/discourse/blob/bf33c287be6081047e56a4ecd77ea4b54a4f5512/plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js#L37-L52

Reply:
Unfortunately I don't think it is possible. Before Glimmer, Ember components can set a `attributeBindings` property but that was removed in Glimmer and for good reasons.